### PR TITLE
add RubyGems.org look and feel

### PR DESCRIPTION
**Pull request by ammancilla**
This PR addresses issue #27

The adoption center now has the RubyGems look and feel in all the public pages:

- Home
- Search
- Gem list
- Gem show
- Recent activity

There are still pending all the pages that requires login. The temporary `sign_in_with_github` option is not working so there is 'no way' to access those pages. **Once we solve #28 I can continue with them.**

### CODE REVIEW

**Comment by ramonh** [comment]
This looks fantastic, thank you @ammancilla! @sebasoga I see you're assigned to this. It's all looking good here. Can I go ahead and merge?

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+      ruby_gem = RubyGem.create!(
+                                  name: Faker::App.name, 
+                                  info: Faker::Lorem.paragraph(2), 
+                                  project_uri: Faker::Internet.url('example.com')
+                                )
+      u.ruby_gems << ruby_gem
+      u.gems_adoptions.create!(ruby_gem_id: ruby_gem.id, description: Faker::Lorem.paragraph)
+    end
+  end
+end
```
Comment (ramonh) [inline db/seeds.rb]: I'm getting an ActiveRecord validation error here when running `db:setup`:
`ActiveRecord::RecordInvalid: Validation failed: Name has already been taken`

Comment (ramonh) [inline db/seeds.rb]: Not sure why it's failing, as Faker should be doing its job...

Comment (ramonh) [inline db/seeds.rb]: Fails on line `12`, dunno if that helps.

Comment (ammancilla) [inline db/seeds.rb]: Hey @ramonh , it is failing because faker is generating the same `App.name` for more than  one `RubyGem`. Im going to fix it,  will let you know when I do.

Comment (ramonh) [inline db/seeds.rb]: Oh ok, good to know, thank you!

Not sure how it's using the same `App.name`, though... Could you please explain to me how? I'm curious :)

Comment (ammancilla) [inline db/seeds.rb]: @ramonh run it now, it won't fail. The `Faker::App.name` function returns a fake name which I'm using for the `name` attribute of the ruby gems. Given the fact that I'm calling that function inside a loop, it seems it is returning the same value more than one time making the uniqueness validation for the name attribute on the RubyGem model fail.

Comment (ramonh) [inline db/seeds.rb]: Nice one, thank you for the explanation @ammancilla!

**Comment by ramonh** [comment]
Wow this looks great! I'd say it's good to merge, what about you @sebasoga?

File: .gitignore [hunk 2]
```diff
@@ -13,6 +13,7 @@ config/database.yml
 # Ignore sqlite3 databases
 **/*.sqlite*
 
-# Ignore and env files
+# Ignore general local files
+Guardfile
```
Comment (sebasoga) [inline .gitignore]: I think this file belongs to this pull request, but the previous one you opened.

File: db/seeds.rb [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+      # Gems
+      ruby_gem = RubyGem.create!(
+                                  name: "#{Faker::App.name}_#{i}_#{j}", 
+                                  info: Faker::Lorem.paragraph(2), 
+                                  project_uri: Faker::Internet.url('example.com')
+                                )
+      u.ruby_gems << ruby_gem
+      u.gems_adoptions.create!(ruby_gem_id: ruby_gem.id, description: Faker::Lorem.paragraph)
+    end
+  end
```
Comment (sebasoga) [inline db/seeds.rb]: I think this can be useful, but I'm not sure having fake gems would be all that useful. Maybe we could ask the RubyGems.org team how they handle populating their local database for development.

I'd rather not introduce a dependency to the project to create seeds, I think you can achieve all of this, by adding static values where possible and using the counter where values need to be unique.

I think this is not really related to adding RubyGems.org look and feel and should be added in another pull request where we can better discuss how to approach seeds.

File: Gemfile [hunk 4]
```diff
@@ -29,7 +30,6 @@ group :production do
 end
 
 # Assets
-gem 'bootstrap-sass', '~> 3.3.5'
```
Comment (sebasoga) [inline Gemfile]: :+1:

File: app/views/adoption_requests/new.html.erb [hunk 5]
```diff
@@ -1,4 +1,5 @@
-<% provide(:title, "Adoption Request Form") %>
+<% page_title! 'Adoption Request Form' %>
```
Comment (sebasoga) [inline app/views/adoption_requests/new.html.erb]: What's the benefit of using the `page_title_helper` gem instead of the `provide` helper?

File: db/seeds.rb (hunk 3 above)
Comment (ammancilla) [inline db/seeds.rb]: Yes, you are right about this. It doesn't belong to this PR. Will remove it.

File: app/views/adoption_requests/new.html.erb (hunk 5 above)
Comment (ammancilla) [inline app/views/adoption_requests/new.html.erb]: Thats the gem RubyGems.org use to manage their page title. I found it easier to add the gem that to replace it.

File: .gitignore (hunk 2 above)
Comment (ammancilla) [inline .gitignore]: Yes, but i needed it to work on this branch. Once we merge the guard PR I will rebase this branch.

**Comment by ammancilla** [comment]
Ping @sebasoga @ramonh

**Comment by ramonh** [comment]
Hey @ammancilla! Apologies for the delay, I'll be finally looking into these :smile:

**Comment by ramonh** [comment]
Thank you so much @ammancilla, this looks fantastic! I'll go ahead and merge.
